distribution/distribution/src/main/kotlin/com/emergetools/distribution/internal/DistributionInternal.kt

```diff
@@ -27,9 +31,11 @@ import okhttp3.Request
 import okhttp3.Response
 import okhttp3.internal.closeQuietly
 import java.io.IOException
+import java.util.concurrent.CompletableFuture
 import kotlin.coroutines.resumeWithException
 
 private const val MANIFEST_TAG_API_KEY = "com.emergetools.distribution.API_KEY"
+private const val MANIFEST_TAG_TAG_KEY = "com.emergetools.distribution.TAG"
```

Comment:
Will this be piped in from the Gradle plugin automatically?

Reply:
I'm planning to do so! Will send you a PR for the gradle plugin side shortly.